### PR TITLE
Add page about yaml validation

### PR DESCRIPTION
**Pull request by helinanever**
- Added a new page under Knowledge base

### CODE REVIEW

File: content/getting-started/yaml.md [hunk 1]
```diff
@@ -19,6 +19,10 @@ Builds can be also started manually by clicking **Start new build** in Codemagic
 
 ## Syntax
 
+{{<notebox>}}
```
Comment (craastad) [inline content/getting-started/yaml.md:22]: I would put this box after the next sentence. First we say it's yaml, then we link to how you validate it.

File: content/knowledge-base/validate-yaml.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+This is an example of using the Codemagic JSON schema for validating codemagic.yaml in Visual Studio Code.
+
+1. In VS Code, make sure you have installed and enabled an extension for YAML language support, e.g. **redhat.vscode-yaml**.
+2. Then navigate to **Code > Preferences > Extensions > Settings > Extensions > YAML**.
+3. Scroll to the **Yaml: Schemas** section and click **Edit in settings.json**.
+4. Append the follwing snippet to the end of the file:
+
+```yaml
+"yaml.schemas": {
```
Comment (craastad) [inline content/knowledge-base/validate-yaml.md:26]: It looks like a `}` is missing for this `{`

File: content/knowledge-base/validate-yaml.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+## How to set up validation in Visual Studio Code
+
+This is an example of using the Codemagic JSON schema for validating codemagic.yaml in Visual Studio Code.
+
+1. In VS Code, make sure you have installed and enabled an extension for YAML language support, e.g. **redhat.vscode-yaml**.
+2. Then navigate to **Code > Preferences > Extensions > Settings > Extensions > YAML**.
+3. Scroll to the **Yaml: Schemas** section and click **Edit in settings.json**.
+4. Append the follwing snippet to the end of the file:
+
+```yaml
```
Comment (BondarenkoStas) [inline content/knowledge-base/validate-yaml.md:25]: don't know if it changes highlighting, but here should be `json`, not `yaml` (and in the script below)

File: content/knowledge-base/validate-yaml.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+* YAML language support in the IDE
+* To configure the IDE to use the Codemagic JSON schema for validation available at [https://static.codemagic.io/codemagic-schema.json](https://static.codemagic.io/codemagic-schema.json)
+
+## How to set up validation in Visual Studio Code
+
+This is an example of using the Codemagic JSON schema for validating codemagic.yaml in Visual Studio Code.
+
+1. In VS Code, make sure you have installed and enabled an extension for YAML language support, e.g. **redhat.vscode-yaml**.
+2. Then navigate to **Code > Preferences > Extensions > Settings > Extensions > YAML**.
```
Comment (BondarenkoStas) [inline content/knowledge-base/validate-yaml.md:21]: extra `Extensions`, should be just
**Code > Preferences > Settings > Extensions > YAML**

**Review by craastad — APPROVED** [review]
Nice :+1:

**Pull request merged**